codeit/classes/Article.js

```diff
@@ -0,0 +1,13 @@
+class Article {
+    constructor(title, content, writer, likeCount) {
+        this.title = title;
+        this.content = content;
+        this.writer = writer;
+        this.likeCount = likeCount;
+        this.createdAt = new Date();  // 생성 시점의 시간 저장
+    }
+
+    like() {
+        return this.likeCount += 1;
+    }
+}
```

Comment:
`like()` 메서드를 통해서만 좋아요 수를 변경하는 게 안전해요.
지금은 `article.likeCount = 100`처럼 외부에서 값이 임의로 변경될 수 있습니다.
아래처럼 private 필드와 getter를 사용하면 캡슐화를 적용할 수 있어요.

```suggestion
class Article {
    #likeCount = 0; 

    constructor(title, content, writer) {
        this.title = title;
        this.content = content;
        this.writer = writer;
        this.createdAt = new Date();
    }

    like() {
        this.#likeCount += 1;
        return this.#likeCount;
    }

    get likeCount() {
        return this.#likeCount;
    }
}
```